.travis.yml

```diff
@@ -5,6 +5,11 @@ services:
 
 before_script:
   - psql -U postgres -c 'CREATE DATABASE chitter_test;'
-
+  - psql -U postgres -d chitter_test -f db/migrations/01-create_peeps_table.sql
+  - psql -U postgres -d chitter_test -f db/migrations/02-update_with_date.sql
+  - psql -U postgres -d chitter_test -f db/migrations/03-create_users_table.sql
+  - psql -U postgres -d chitter_test -f db/migrations/04-update_peeps_wit_peep_name.sql
+  - psql -U postgres -c 'CREATE DATABASE chitter;'
```

Comment:
I'm not sure why this line is needed? I thought Travis was only testing, so would only need the test database?